src/DynamoCoreWpf/ViewModels/Core/GeometryScalingViewModel.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Collections.Generic;
+using Dynamo.Core;
+using Dynamo.ViewModels;
+
+namespace ViewModels.Core
+{
+    /// <summary>
+    /// This class will be contain information about the current Geometry Scale selected in the Dynamo Workspace
+    /// </summary>
+    public class GeometryScalingViewModel : NotificationObject
```

Comment:
Why is this public is it needed for binding?

Reply:
Modified to be internal in the next PR: https://github.com/DynamoDS/Dynamo/pull/13734